
+++ //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/userspace/private/apps/hndrouter/Makefile
--- //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/userspace/private/apps/hndrouter/Makefile

@@ -18,6 +18,7 @@
 export WLMAIN = main/src
 export BUILD_MCAST_PROXY
 export BUILD_BSTREAM_IQOS
+export BUILD_WLAN_TESTBED_AP
 
 ifneq ($(strip $(BCA_CPEROUTER)),)
 -include $(HNDAPPS_DIR)/.config
